main.js

```diff
@@ -251,9 +241,7 @@ class DataEntryInstance extends InstanceBase {
 				width: 6,
 				default: '*',
 				useVariables: true,
-				isVisible: (opt) => {
-					return !!opt && typeof opt.formattype === 'string' && opt.formattype.startsWith('ecma')
-				},
+				isVisibleExpression: `$(options:formattype) == 'ecmaauto' || $(options:formattype) == 'ecmastring' || $(options:formattype) == 'ecmanumber' || $(options:formattype) == 'ecmadate' || $(options:formattype) == 'ecmatime' || $(options:formattype) == 'ecmabool'`,
```

Comment:
Perhaps it could be possible to use `includes($(options:formattype), "ecma")`, to make this a little cleaner?